edgelet/iotedge/src/check/mod.rs

```diff
@@ -890,4 +902,64 @@ mod tests {
             ),
         }
     }
+
+    #[test]
+    fn pickup_proxy_uri_from_the_right_place() {
+        //Setup the environment
+        let env_proxy_uri = "https://environment:123";
+        std::env::set_var("HTTPS_PROXY", env_proxy_uri);
```

Comment:
Test "https_proxy" (lowercase) env var too?

Reply:
Yes, needs to check both, like the old code used to.